May newsletters with monthly updates (#273)
readthedocs/ethicalads.io: content/posts/newsletter-may-2022.md

Title: EthicalAds Newsletter - May 2022
Date: June 9, 2022
description: Updates from May, including stats and commentary on our progress as we build EthicalAds
tags: newsletter, community
authors: David Fischer
image: /images/posts/2022-may-newsletter.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@by_syeoni?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Suhyeon Choi</a> on <a href="https://unsplash.com/s/photos/may?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

## New features

These are the major features we released in the last month:

* In May, we rolled out the ability for publishers to show their own fallback house ads
  when we don't have a paid ad to show.

>   We will have more details on this but this was one of the most requested features by publishers.

Reviewer: ```suggestion
You can access it on `Fallback Ads` menu in your publisher dashboard, if you want to try it out.```

* After seeing the success of the small onboarding improvements in our ad server,
  we reworked a number of our standard customer support responses
  to help get new publishers and advertisers on the platform as easily and quickly as possible.

> * Our keyword analyzer is now spidering automatically across the most visited pages across our network
>   to help us better understand the content where we show ads and show the most relevant ad.

Reviewer: ```suggestion
* Our keyword analyzer is now spidering across the most visited pages in our network.
  This helps us better understand the content where we show ads, curate topics, and show the most relevant ad.
```

* We did some initial work on building more targeting and pricing information into our ad server.
  Specifically, having multiple preset regions and topics based on what advertisers frequently buy.
* On the marketing front, we produced a page on how we are

>   [different from other developer ad networks like CarbonAds]({filename}../pages/vs-carbon-ads.md?ref=newsletter)

Reviewer: 😍
